src/main/java/com/service/QuoteService.java

```diff
@@ -1,60 +1,68 @@
-package com.service;
-
-import java.util.List;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
-
-import com.dao.QuoteRepository;
-import com.model.Quote;
-
-
-/**
- * 
- * QuoteService class for accessing the dao layer.
- * 
- * @author Neutralizer
- *
- */
-@Service
-public class QuoteService {
-
-	@Autowired
-    private QuoteRepository quoteRepository;
-
-
-	/**
-	 * Return all quotes from repo.
-	 * @return All quotes
-	 */
-    public List<Quote> getAllQuotes() {
-        return quoteRepository.getAllQuotes();
-    }
-
-    /**
-     * 
+package com.service;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.dao.QuoteRepository;
+import com.model.Quote;
+
+
+/**
+ * 
+ * QuoteService class for accessing the dao layer.
```

Comment:
The service layer is for business logic, it's not specifically for accessing the dao layer.

Reply:
Fixed in neutralizer/fixes-for-review-for-model-expanding-branch